src/renderer.js

```diff
@@ -116,6 +116,9 @@ export function toArray(text, options = {}) {
     while (previousTextWithoutAsciiAliases !== textWithoutAsciiAliases) {
       previousTextWithoutAsciiAliases = textWithoutAsciiAliases;
       textWithoutAsciiAliases = textWithoutAsciiAliases.replace(
+        aliasesRegex,
+        replaceAliases
```

Comment:
This replace here doesn't make a lot of sense due to the context on the function. It should be executed on `text` before the call to `replaceAllAsciiAliases` (see next comment).

Reply:
(see reply to next comment)